README.md

```diff
@@ -45,6 +45,8 @@ From source:
 
 ### Input
 
+The received `message` is an array of numbers corresponding to the midi bytes: `[status, data1, data2]`. Detailed information on the MIDI format can be found [on this page](https://www.cs.cf.ac.uk/Dave/Multimedia/node158.html) for example.
```

Comment:
This is great. Can we capitalize MIDI though?

Though I'm wondering if it would be more helpful if we split it up and call out the format in a comment on the callback (~line 62) but leave the general info link up here. What do you think about that?